packages/node/src/node.ts

```diff
@@ -32,13 +32,25 @@ export default class Node {
   constructor(
     privateKey: string,
     private readonly messagingService: IMessagingService,
-    private readonly storeService: IStoreService
+    private readonly storeService: IStoreService,
+    nodeConfig: NodeConfig
   ) {
     this.signer = new ethers.utils.SigningKey(privateKey);
     this.incoming = new EventEmitter();
     this.outgoing = new EventEmitter();
-    this.registerListeners();
-    this.registerConnection();
+    this.channels = new Channels(
+      this.signer.address,
+      this.storeService,
+      // naive, account-based multisig indexing
+      `${nodeConfig.MULTISIG_KEY_PREFIX}/${this.signer.address}`
+    );
+    this.registerMessagingConnection();
+    new MethodHandler(
```

Comment:
@cf19drofxots This is a bit weird. Since all of the `MethodHandler` class members are private, maybe it should be a function instead of a class?

Reply:
This is also stale :P

Not all its method are private now.